### PR TITLE
Test pull request.

### PR DESCRIPTION
**Pull request by ngold**
**I'm testing a GIT pull request**

### CODE REVIEW

File: Armstrong.WebClient/Program.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
-builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddScoped(sp => new HttpClient { 
+    BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) 
+});
```
Comment (xdcarp) [inline Armstrong.WebClient/Program.cs]: Make this a unique line of code

File: Armstrong.WebClient/Program.cs [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
 
-await builder.Build().RunAsync();
+await builder.Build().RunAsync();
```
Comment (xdcarp) [inline Armstrong.WebClient/Program.cs]: Why do you have this rare of (end of file)

**Review by xdcarp — CHANGES_REQUESTED** [review]

File: Armstrong.WebClient/Program.cs (hunk 1 above)
Comment (ngold) [reply Armstrong.WebClient/Program.cs]: I fixed this issue

File: Armstrong.WebClient/Program.cs (hunk 2 above)
Comment (ngold) [reply Armstrong.WebClient/Program.cs]: Fixed

**Review by xdcarp — APPROVED** [review]
Approved

**Pull request merged**